src/etc/checkstyle-suppressions.xml

```diff
@@ -22,6 +22,6 @@
 
 <suppressions>
     <suppress
-            checks="JavadocStyleCheck|JavadocMethodCheck|JavadocTypeCheck|JavadocVariableCheck|LineLength|FinalParameters|AvoidStarImport"
+            checks="JavadocStyleCheck|JavadocMethodCheck|JavadocTypeCheck|JavadocVariableCheck|LineLength|FinalParameters|AvoidStarImport|AvoidInlineConditionals"
```

Comment:
While in some cases inline conditionals are hard to read, there are quite a few cases where they are the right solution. I'd like not to have a hard rule on them but rely on code review to spot the hard to read ones.